nftproject/contracts/NFT.vy

```diff
@@ -8,23 +8,25 @@ implements: ERC721
 
 ############ ERC-165 #############
 # @dev Static list of supported ERC165 interface ids
-SUPPORTED_INTERFACES: constant(bytes4[4]) = [
-    0x01ffc9a7,  # ERC165 interface ID of ERC165
-    0x80ac58cd,  # ERC165 interface ID of ERC721
-    0x5b5e139f,  # ERC165 interface ID of ERC721 Metadata Extension
-    0x5604e225,  # ERC165 interface ID of ERC4494
+SUPPORTED_INTERFACES: constant(bytes4[6]) = [
+    0x01ffc9a7,  # interface ID of ERC165
+    0x80ac58cd,  # interface ID of ERC721
+    0x5b5e139f,  # interface ID of ERC721 Metadata Extension
+    0x5604e225,  # interface ID of ERC 4494
+    0x150b7a02,  # interface ID of ERC721 Token Receiever
```

Comment:
This contract would not implement this interface ID, would suggest to make a separate test contract (under `contracts/test/TokenReceiver.vy`) for testing purposes